### ISSUE TITLE
This plugin overrides custom theme

### ISSUE THREAD

**Issue by xFlatlinex** [ChrisRM/material-theme-jetbrains#20]
I'm having the same issues as #12 and #16, but these are just simple configuration issues, so I decided to modify the theme into a custom one using the "Save as..." feature on "Colors & Fonts".  
That works well except that the next time I open PhpStorm it doesn't load my custom theme, but the one that comes with this plugin instead, forcing me to go to the config and choose my custom theme again.

So basically, I cannot use a different theme than the one that comes with this plugin without changing it manually every time I open PhpStorm.  
I don't know if this is a problem with this plugin or just how PhpStorm works with themes in plugins, hopefully is a problem with the plugin itself, so it can be solved :)

**Comment by cryode** [comment]
+1 on PhpStorm not remembering the custom theme after restarting.
